mini_racer.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |spec|
     "source_code_uri"   => "https://github.com/discourse/mini_racer/tree/v#{spec.version}",
   }
 
-  spec.files         = Dir["lib/**/*", "ext/**/*", "README.md", "LICENSE.txt", "CHANGELOG", "CODE_OF_CONDUCT.md"]
+  spec.files         = Dir["lib/**/*", "ext/**/*", "README.md", "LICENSE.txt", "CHANGELOG", "CODE_OF_CONDUCT.md"].reject { |fn| File.fnmatch?("**/*.{bundle,c,cc}", fn, File::FNM_EXTGLOB) }
```

Comment:
Shouldn't this rather simply read `lib/**/*.rb`

Reply:
Also I think we need to keep the C files, otherwise the extension won't build once packaged as a gem.

Reply:
erm, of course🤦 thanks

but one more possible item to improve CI builds, because it was all green.